content/applications/finance/fiscal_localizations/canada.rst

```diff
@@ -0,0 +1,311 @@
+
+.. seealso::
+   - :doc:`../accounting/get_started/chart_of_accounts`
+   - :doc:`../accounting/get_started/cheat_sheet`
+
+.. _l10n_ca/fiscal-positions:
+
+Fiscal positions
+================
+
+Canadian tax rates and taxable items vary by province and territory. Default fiscal positions are
+automatically created when the Odoo **Accounting** application is installed. To manage or configure
+additional fiscal positions, navigate to :menuselection:`Accounting --> Configuration --> Fiscal
+Positions`.
+
+The following fiscal positions are available by default:
+
+- :guilabel:`Alberta (AB)`
+- :guilabel:`British Colombia (BC)`
+- :guilabel:`Manitoba (MB)`
+- :guilabel:`New Brunswick (NB)`
+- :guilabel:`Newfoundland and Labrador (NL)`
+- :guilabel:`Nova Scotia (NS)`
+- :guilabel:`Northwest Territories (NT)`
+- :guilabel:`Nunavut (NU)`
+- :guilabel:`Ontario (ON)`
+- :guilabel:`Prince Edward Islands (PE)`
+- :guilabel:`Quebec (QC)`
+- :guilabel:`Saskatchewan (SK)`
+- :guilabel:`Yukon (YT)`
+- :guilabel:`International (INTL)`
+
+.. image:: canada/l10n-ca-fiscal-positions.png
+   :alt: The default fiscal positions for the Canada localization in Odoo Accounting.
+
+.. note::
+   When considering what taxes to be applied, it is the province where the delivery occurs that
+   matters. Therefore delivery is the responsibility of the vendor and is accounted for at the
+   customer location.
+
+.. example::
+   - A delivery is made to a customer from another province.
+        Set the fiscal position on the customer's record to the province of the customer.
+   - A customer from another province comes to pick up products.
+        No fiscal position should be set on the customer's record.
+   - An international vendor doesn't charge any tax, but taxes are charged by the customs broker.
+        Set the fiscal position on the vendor's record to *International*.
+   - An international vendor charges provincial tax.
+        Set the fiscal position on the vendor's record to your position.
+
+.. seealso::
+   :doc:`../accounting/taxes/fiscal_positions`
+
+.. _l10n_ca/taxes:
+
+Taxes
+=====
+
+In Canada, tax rates and what is considered taxable vary by province and territory. Default *Sales*
+and *Purchases* taxes are created automatically when the Odoo **Accounting** application is
+installed. To manage existing or configure additional taxes, navigate to
+:menuselection:`Accounting --> Configuration --> Taxes`.
+
+.. _l10n_ca/taxes-avatax:
+
+AvaTax
+------
+
+**Avalara AvaTax** is a cloud-based tax calculation and compliance software that integrates with
+Odoo for several localizations. Integrating AvaTax with Odoo provides real-time and region-specific
+tax calculations when items are sold, purchased, and invoiced in the database.
+
+.. important::
+   AvaTax is available for integration with databases/companies that have locations in Canada and
+   the United States. Reference the :ref:`avatax/fiscal_country` documentation for more information.
```

Comment:
important detail to confirm
is the integration available for companies that **only** have locations in CA? Or is a US location required too?
```suggestion
   AvaTax is available for integration with databases/companies that have locations in Canada and/or
   the United States. Reference the :ref:`avatax/fiscal_country` documentation for more information.
```